File: 백준/Bronze/10810. 공 넣기/공 넣기.js
let fs = require("fs");
let input = fs.readFileSync("/dev/stdin").toString().trim().split("\n");

const N = +input[0].split(" ")[0];
const M = +input[0].split(" ")[1];
let arr = new Array(N).fill(0);
for (let i = 1; i < input.length; i++) {
  const perArr = input[i].split(" ");
  for (let j = +perArr[0] - 1; j < +perArr[1]; j++) {
    arr[j] = +perArr[2];
  }
}
console.log(arr.join(" "));

File: 백준/Bronze/10810. 공 넣기/공 넣기.py
import sys
input = sys.stdin.readline
N, M = list(map(int, input().split(" ")))
arr = [0 for _ in range(N)]
for _ in range(M):
    i,j,k = map(int, input().split(" "))
    for l in range(i, j+1):
        arr[l-1] = k
for l in range(N):
    print(arr[l], end=" ")